docs/theming/high-contrast-mode.md

```diff
@@ -0,0 +1,245 @@
+
+<Tabs groupId="framework" defaultValue="angular" values={[{ value: 'angular', label: 'Angular' }, { value: 'javascript', label: 'Javascript' }, { value: 'react', label: 'React' }, { value: 'vue', label: 'Vue' }]}>
+
+<TabItem value="angular">
+
+```css
+@import '@ionic/angular/css/themes/high-contrast.system.css';
+@import '@ionic/angular/css/themes/high-contrast-dark.system.css';
+```
+
+</TabItem>
+<TabItem value="javascript">
+
+```ts
+import '@ionic/core/css/themes/high-contrast.system.css';
+import '@ionic/core/css/themes/high-contrast-dark.system.css';
+```
+
+</TabItem>
+<TabItem value="react">
+
+```tsx
+import '@ionic/react/css/themes/high-contrast.system.css';
+import '@ionic/react/css/themes/high-contrast-dark.system.css';
+```
+
+</TabItem>
+<TabItem value="vue">
+
+```ts
+import '@ionic/vue/css/themes/high-contrast.system.css';
+import '@ionic/vue/css/themes/high-contrast-dark.system.css';
+```
+
+</TabItem>
+
+</Tabs>
+
+This approach activates the high contrast theme when the [CSS media query for `prefers-contrast`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast) is `more`. The `prefers-contrast` media query is supported by [all modern browsers](https://caniuse.com/?search=prefers-contrast). If support for an older browser is required, we recommend using the [class](#class) approach.
+
+The following example uses the system settings to decide when to show high contrast mode.
+
+:::info
+Not sure how to change the system settings? Here's how to enable high contrast mode on [Windows 11](hhttps://support.microsoft.com/en-us/windows/turn-high-contrast-mode-on-or-off-in-windows-909e9d89-a0f9-a3a9-b993-7a6dcee85025) and on [macOS](https://support.apple.com/guide/mac-help/change-display-settings-for-accessibility-unac089/mac).
+:::
+
+import SystemHighContrastMode from '@site/static/usage/v8/theming/system-high-contrast-mode/index.md';
+
+<SystemHighContrastMode />
+
+:::caution
+The high contrast light theme must be imported after [core.css](../layout/global-stylesheets#corecss), and the
+high contrast dark theme must be imported after `dark.system.css`. Otherwise, the standard contrast theme will take priority.
+:::
+
+### Class
+
+While the previous approaches are excellent for enabling the high contrast theme through file imports alone, there are scenarios where you may need more control over its application. In cases where you need to apply the high contrast theme conditionally, such as through a toggle, or if you want to extend the functionality based on system settings, we provide a high contrast theme class file. This file applies the high contrast theme when a specific class is added to an app. Importing the following stylesheets into the appropriate file will provide the necessary styles for using the high contrast theme with the class:
```

Comment:
```suggestion
While the previous approaches are excellent for enabling the high contrast theme through file imports alone, there are scenarios where you may need more control over where it is applied. In cases where you need to apply the high contrast theme conditionally, such as through a toggle, or if you want to extend the functionality based on system settings, we provide a high contrast theme class file. This file applies the high contrast theme when a specific class is added to an app. Importing the following stylesheets into the appropriate file will provide the necessary styles for using the high contrast theme with the class:
```

~ Optional. I think using "application" can get confusing vs. the context of an app someone is building.